cmd/eksctl-anywhere/cmd/version.go

```diff
@@ -8,20 +8,44 @@ import (
 	"github.com/aws/eks-anywhere/pkg/version"
 )
 
+type versionOptions struct {
+	output string
+}
+
+var vo = &versionOptions{}
+
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Get the eksctl anywhere version",
 	Long:  "This command prints the version of eksctl anywhere",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return printVersion()
+		return vo.printVersion()
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(versionCmd)
+	versionCmd.Flags().StringVarP(&vo.output, "output", "o", "", "specifies the output format (valid option: json)")
 }
 
-func printVersion() error {
-	fmt.Println(version.Get().GitVersion)
+func (vo *versionOptions) printVersion() error {
+	switch vo.output {
+	case "":
+		versionInfo, err := version.GetFullVersionInfo()
+		fmt.Printf("Version: %s\n", versionInfo.GitVersion)
+		if err != nil {
+			return fmt.Errorf("error getting bundle manifest URL for version %s: %v", versionInfo, err)
+		}
+		fmt.Printf("Bundle manifest URL: %s\n", versionInfo.BundleManifestURL)
```

Comment:
the flux example has all components printed in lowercase and without any spaces. This is advantageous when parsing with command line tools. Might be something to consider here too. 

```bash
version: v0.0.0-dev
bundle-url: https://dev-release-assets.eks-anywhere.model-rocket.aws.dev/bundle-release.yaml
```

Reply:
I made the version output look similar to `kubectl version` whose default format includes capitalization and spaces. I have included an `output` flag which allows printing the version in JSON or YAML, and the JSON tags used for the Info struct is easily parseable (`version`, `bundleManifestURL`, etc)